### ISSUE TITLE
textStyles warning

### ISSUE THREAD

**Issue by TeRMiNaTe** [mehulmpt/react-native-hr-component#1]
If you use StyleSheet like for example `textStyles={styles.orText} ` you get this error:
`Failed prop type: Invalid prop 'textStyles' of type 'number' supplied to 'Hr', expected 'object'.`

It seems to work OK, but the warning could be fixed probably. Here is my sample StyleSheet:

`const styles = StyleSheet.create({
	orText: {
		color: '#69b800',
	},
});`

**Comment by mehulmpt** [comment]
Fixed in v2.0.0 released

**Comment by csamyphew** [comment]
The error occur again in v2.0.2

**Comment by csamyphew** [comment]
I used Stylesheet as `textStyles={styles.hrText}`
```
const styles = StyleSheet.create({ 
  hrText:{
    fontFamily: 'Muli',
    color: colors.lightBlueGrey,
    fontSize: 15,
    textAlign: 'center',
  }
})
```
I got error of `Failed prop type: Invalid props.textStyles key 'fontFamily' supplied to 'Hr'`. And the style is not applying too.

**Comment by csamyphew** [comment]
The error still appear in v2.0.2

**Comment by mehulmpt** [comment]
@csamyphew please provide a way to duplicate the behavior

**Comment by martin9908** [comment]
I am experiencing the same error
here is a snippet of my code

`<Hr lineColor="#CECECE" width={2} text=" OR" textStyles={styles.divideStyle}/>`

`divideStyle: {
    color: "#CE2D2B",
    fontFamily: "Lato-Black",
    fontSize: "10@s",
    letterSpacing: "2@s",
    textAlign: "center"
  },`

I am using this release version: "react-native-hr-component": "^2.0.2",

![image](https://user-images.githubusercontent.com/14294449/61779413-322ebe00-ae33-11e9-8a7e-ff6e2ffd40dc.png)

**Comment by mehulmpt** [comment]
Working on it

**Comment by mehulmpt** [comment]
Should be patched in release v2.0.3. It's a patch release so nothing would break. 

`npm i react-native-hr-component@2.0.3 --save`
